doc/authentication-backends/JWT-authentication-module.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+JWT authentication backend can verify JSON Web Tokens provided by the clients.
+A wide range of signature algorithms is supported, including those using public key cryptography.
+Not only the signature is checked, but also the validity of `exp` (expired token is rejected), `iat` (token must be issued in the past) and `nbf` (token might be invalid *yet*) parameters.
+
+Requires SASL PLAIN method.
+
+## Configuration options
+
+* **jwt_secret_source**
+    * **Description:** A path to a file or environment variable, which contents will be used as a JWT secret.
+    * **Warning:** A direct path to a file is read only once during startup, a path in the environment variable is read on every auth request.
+    * **Value:** string, e.g. `/etc/secrets/jwt` or `{env, "env-variable-name"}`
+    * **Default:** none, either `jwt_secret_source` or `jwt_secret` must be set
+
+* **jwt_secret**
+    * **Description:** A binary with a JWT secret. This options is ignored and overwritten, if `jwt_secret_source` is defined.
```

Comment:
This option is ignored and overwritten, if `jwt_secret_source` is defined.
or
 These options are ignored and overwritten, if `jwt_secret_source` is defined.